docs/pages/file-format-coordinates.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Coordinate Systems
+section: file-format
+chapter: Concepts
+permalink: /file-format/coordinate-systems
+
+order: 102
+excerpt: How Sketch stores coordinate information in page.json
+---
+
+When creating shapes, whether it's in Sketch, or in the API, you use define sizes naturally.
```

Comment:
The entire page is only about the points of a path of a shape but it's only mentioned near the bottom of the page. 

Also when you defines the points of a ShapePath in the API, you need to use the normalised coordinates, not what you call the natural sizes.

Reply:
@mathieudutour - I've updated the page name/contents to reflect this

Reply:
err, I think you removed the page completely haha

Reply:
:-) It's back now